src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -86,6 +87,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case ListCommand.COMMAND_WORD:
             return new ListCommand();
 
+        case ViewTagCommand.COMMAND_WORD:
+            return new ViewTagCommand();
```

Comment:
if possible can add a test in AddressBookParserTest to check if this branch executes? for regression testing purposes

Reply:
tested added